custom_schemas/custom_process.yml

```diff
@@ -0,0 +1,523 @@
+---
+- name: process
+  title: Process
+  group: 2
+  short: These fields contain information about a process.
+  description: >
+    These fields contain information about a process.
+
+    These fields can help you correlate metrics information with a process id/name
+    from a log message.  The `process.pid` often stays in the metric itself and is
+    copied to the global field for correlation.
+  reusable:
+    top_level: true
+    expected:
+      - parent
+      - target
+  type: group
+  fields:
+    - name: unique_pid
+      level: custom
+      type: keyword
+      description: >
+        Unique process id.
+
+    - name: is_sensor
```

Comment:
Consider changing to `is_endpoint`.  Sensor is an SMP term not an Elastic Endpoint term.